lambda_function/outputs.tf

```diff
@@ -1,3 +1,3 @@
-output "lambda_arn" {
-  value = "${aws_lambda_function.lambda.arn}"
-}
+output "arn" {
```

Comment:
Can you add the role arn as an output as well please

Reply:
Done in 6524218b9d4d5fdbee94708f8d21c667a1d0d074